modular_skyrat/modules/customization/controllers/subsystem/alt_titles.dm

```diff
@@ -0,0 +1,35 @@
+//SKYRAT MODULE CUSTOMIZATION
+// Make a seperate subsystem so it doesn't conflict with the actual job one.
+
+SUBSYSTEM_DEF(altjob)
+	name = "Alt Jobs"
+	init_order = INIT_ORDER_JOBS
+	flags = SS_NO_FIRE
+
+/datum/controller/subsystem/altjob/Initialize(timeofday)
+	RegisterSignal(SSdcs, COMSIG_GLOB_JOB_AFTER_SPAWN, .proc/on_job_after_spawn)
+	return ..(timeofday)
+
+/datum/controller/subsystem/altjob/proc/on_job_after_spawn(datum/source, datum/job/job, mob/living/spawned, client/player_client)
+	SIGNAL_HANDLER
+
+	var/alt_title = player_client.prefs.alt_job_titles[job.title]
```

Comment:
Add null-checks because we never know if a client couldn't be null at one point. You know how BYOND is.